engine-tests/src/main/java/org/terasology/HeadlessEnvironment.java

```diff
@@ -132,15 +116,15 @@
  */
 public class HeadlessEnvironment extends Environment {
 
-    protected static final WorldTime WORLD_TIME = new WorldTimeImpl();
+    private static final WorldTime WORLD_TIME = new WorldTimeImpl();
     private static final Logger logger = LoggerFactory.getLogger(HeadlessEnvironment.class);
 
     /**
      * Setup a headless ( = no graphics ) environment
      *
      * @param modules a set of module names that should be loaded (latest version)
      */
-    public HeadlessEnvironment(Name... modules) {
+    HeadlessEnvironment(Name... modules) {
```

Comment:
The removal here of `public` breaks the NameGenerator module - generally the `HeadlessEnvironment` is available for other things to use, an IDE might just not think it is in-use if you lack whatever module or facade in your workspace.